scripts/install-snapshot.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash -e
+
```

Comment:
please add some comment in the header of this script, document what it steps it does